libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.ts

```diff
@@ -218,12 +219,15 @@ export class DhWholesaleSettlementsReportsTabsBalanceComponent
       message: this.transloco.translate('wholesale.settlementReports.downloadStart'),
     });
 
+    const startPeriod = this.tryAddOneDay(this.searchForm.controls.period.value?.start);
```

Comment:
Question: I understand what you're doing, however my question is if we're interested in calling the API if the periods  are not set? (the empty string fallback).

Reply:
Good question. The previous code had this logic:

this.searchForm.controls.period.value?.start ?? ''